middlewares/search.middleware.js

```diff
@@ -62,6 +61,48 @@ async function executeQuery(req, res, next) {
     return next();
 }
 
+/**
+ * 
+ * @param {} req 
+ * @param {*} res 
+ * @param {*} next
+ * 
+ * @returns 
+ */
+async function executeStatusAction(req, res, next) {
+    req.body.results = await Services.Search.executeStatusAction(req.body.model,
+        req.body.q,
+        req.body.page,
+        req.body.limit,
+        req.body.sort,
+        req.body.sort_by,
+        req.body.update,
+        req.body.expand
+    );
+    return next();
+}
+
+/**
+ * 
+ * @param {} req 
+ * @param {*} res 
+ * @param {*} next
+ * 
+ * @returns 
+ */
+async function executeEmailAction(req, res, next) {
```

Comment:
How do I trigger different types of emails? Is this restricted just to status emails?

Reply:
Does the executeEmailAction service seem to only handle sending status updates? It's a little bit misleading right now. I would consider renaming it unless there are plans to have this function send emails for generic bulk actions that act on the result of a search. Or something like that.